backend/controllers/appointments/requestBooking.js

```diff
@@ -1,64 +1,167 @@
 const moment = require('moment')
 const { buildDateTime, DATE_TIME_FORMAT_SPEC, DAY_MONTH_YEAR, Time } = require('../../../src/dateHelpers')
 const { serviceUnavailableMessage } = require('../../common-messages')
-const { validateComments, validateDate, validateStartEndTime } = require('../../shared/appointmentConstants')
+const { validateComments } = require('../../shared/appointmentConstants')
 const {
   notifications: { requestBookingCourtTemplateId },
 } = require('../../config')
 
-const requestBookingFactory = ({ logError, notifyClient }) => {
+const prisonsContactConfig = [{ text: 'HMP Wandsworth', value: 'dominic.bull@digital.justice.gov.uk' }]
```

Comment:
This is for Dom to do some testing I assume?

Reply:
It's just temporary, need to sort the config out in another PR.